cmd/mapcidr/main.go

```diff
@@ -107,8 +107,8 @@ func ParseOptions() *Options {
 
 	// Miscellaneous
 	flagSet.CreateGroup("miscellaneous", "Miscellaneous",
-		flagSet.BoolVarP(&options.SortAscending, "sort", "s", false, "Sort input IPs/CIDRs in ascending order"),
-		flagSet.BoolVarP(&options.SortDescending, "sort-reverse", "sr", false, "Sort input IPs/CIDRs in descending order"),
+		flagSet.BoolVarP(&options.SortAscending, "sort", "s", false, "Sort input IPs in ascending order"),
```

Comment:
it seems to me that both CIDRs and IPs are supported?

Reply:
Sorry, I'm not very good with git, I've been using my previous code to make changes, which caused some of the code to be overwritten with past versions.

Now, I've reverted the changes and committed

Reply:
This commit is only about match and filter.